src/webview/android/main_pipe.rs

```diff
@@ -183,9 +183,15 @@ impl MainPipe<'_> {
           }
         }
         WebViewMessage::Jni(f) => {
-          if let Some(webview) = &self.webview {
-            f(env, activity, webview.as_obj());
-          }
+          f(
+            env,
+            activity,
+            self
+              .webview
+              .as_ref()
+              .map(|w| w.as_obj().clone())
+              .unwrap_or_default(),
+          );
```

Comment:
Could we keep previous if let style? It's more readable IMHO.

Reply:
done @wusyong